node/src/neighborhood/mod.rs

```diff
@@ -561,7 +562,7 @@ impl Neighborhood {
                     self.user_exit_preferences.locations_opt.clone()
                 {
                     for exit_location in &exit_locations_by_priority {
-                        self.enrich_exit_countries(&exit_location.country_codes);
+                        self.enrich_exit_countries_returns_missing(&exit_location.country_codes);
```

Comment:
Takhle je to takove zvlastni. Jako by ta funkce mela vracet bool. Je to proto, ze jsi nepouzil rozkazovaci zpusob, tam je to pak jasne. 
Takze si myslim, ze jistotou je "enrich_exit_countries_and_return_missing".